configs/detection/slowfast/slowfast_kinetics400-pretrained-r50_8xb16-4x16x1-8e_multisports-rgb.py

```diff
@@ -0,0 +1,135 @@
+_base_ = [
+    '../slowonly/slowonly_kinetics400-pretrained-r50_8xb16-4x16x1-8e_multisports-rgb.py'  # noqa: E501
+]
+
+url = ('https://download.openmmlab.com/mmaction/recognition/slowfast/'
+       'slowfast_r50_4x16x1_256e_kinetics400_rgb/'
+       'slowfast_r50_4x16x1_256e_kinetics400_rgb_20200704-bcde7ed7.pth')
+num_classes = 66
+model = dict(
+    init_cfg=dict(type='Pretrained', checkpoint=url),
+    backbone=dict(
+        _delete_=True,
+        type='ResNet3dSlowFast',
```

Comment:
may be better to use `type='mmaction.ResNet3dSlowFast',`

Reply:
that's good